modules/ROOT/pages/_partials/api-mgr.adoc

```diff
@@ -182,12 +182,20 @@ In addition to the default flag `--help`, this command accepts the following fla
 | The CloudHub Gateway version
 | `--gatewayVersion: 9.9.9.9  643404`
 
+| `--javaVersion`
+| Gateway Java version
+| `--javaVersion 17 643404`
+
+| `--releaseChannel`
+| Release channel
+| `--releaseChannel EDGE 643404`
```

Comment:
Can you specify for which deployment types is it used?

Reply:
sure!